FreeRTOS-Plus/Demo/MQTT_Mutual_Auth_Qemu_mps2/main.c

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
Is this duplicated from other demos?

Reply:
Yes, from the freertos + tcp qemu mps2 demo with main_echo_client_tasks updated to be vMainMQTTClientTasks